### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by lucasdigital**
Will definitely need some adjustments.

### CODE REVIEW

File: docs/network.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+id: network-requirements
+title: Network Requirements
+sidebar_label: Network requirements
+---
+
+Set up your network before installing LiveData Migrator (LDM).
```
Comment (stepgerner) [inline docs/network.md:7]: Something about this as the first sentence feels really weird and detached to me, it's jarring and I can't put my finger on why. I know that you're trying to go for the consistent tense of purely telling users what to do, but...

Maybe "Before installing LiveData Migrator, you must set up your network to accommodate its data transference." Or something. That's a bit bad, but maybe you get what I mean?

File: docs/prereqs.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ Ready to start? Check the prerequisites and then follow these steps to get ready
 
 * Proficiency with intended target storage technologies, such as ADLS Gen 2, GCS, HDFS, and Amazon Simple Storage Service (S3).
 * For Amazon Web Services, this includes:
-  - Knowledge of AWS Marketplace, Amazon Elastic Compute Cloud (Amazon EC2) Console, Amazon Simple Storage Service (Amazon S3), and AWS Command Line Tool.
-  - Specifically, starting, stopping, and terminating AWS instances, understanding any storage persistence and related costs.
-  - Ability to monitor and troubleshoot AWS services.
+  * Knowledge of AWS Marketplace, Amazon Elastic Compute Cloud (Amazon EC2) Console, Amazon Simple Storage Service (Amazon S3), and AWS Command Line Tool.
+  * Specifically, starting, stopping, and terminating AWS instances, understanding any storage persistence and related costs.
```
Comment (stepgerner) [inline docs/prereqs.md:31]: Just as it feels like clearer English in the context of the preceding content, I'd say:
"Knowledge of the AWS Marketplace..."
"The ability to start, stop and terminate AWS instances and an understanding of any storage persistence and related costs."
"An understanding of how to monitor and troubleshoot AWS services."

**Review by stepgerner — COMMENTED** [review]
Just two comments, reads really well otherwise.

File: docs/prereqs.md (hunk 2 above)
Comment (benradstone) [reply docs/prereqs.md:31]: I didn't write any of this.

Comment (lucasdigital) [reply docs/prereqs.md:31]: I think that came right out of the AWS manual. ;-)

File: docs/network.md (hunk 1 above)
Comment (benradstone) [reply docs/network.md:7]: I'll adjust it slightly.

**Pull request closed without merging**